README.md

```diff
@@ -10,6 +10,10 @@ You're welcome to try it, and let us know your feedback in the issues!
 This is Honeycomb's Distribution of OpenTelemetry for Node.js.
 It makes getting started with OpenTelemetry and Honeycomb easier!
 
+## OpenTelemetry Packages
+
+The distro was built with [v1.17.1](https://github.com/open-telemetry/opentelemetry-js/releases/tag/v1.17.1) and [v0.44.0](https://github.com/open-telemetry/opentelemetry-js/releases/tag/experimental%2Fv0.44.0) of OpenTelemetry JS packages.
```

Comment:
We should have this match the [versions of the latest release](https://github.com/honeycombio/honeycomb-opentelemetry-node/blob/v0.5.0/package.json), not main. Then in releasing the new version we'll update.

Also maybe we can bullet this like we have in the other distros, and note what version of auto-instr it's compatible with based on the example app

Reply:
done in https://github.com/honeycombio/honeycomb-opentelemetry-node/pull/241/commits/d7b76a8fee14c3d85d45a602356b2121760a8d1b